### PR TITLE
[Main] Release 1.1.0

### PR DESCRIPTION
**Pull request by Moritz-Alexander-Kern**
This is housekeeping for release 1.1.0.

### CODE REVIEW

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

File: doc/release_notes.rst [hunk 1]
```diff
@@ -3,6 +3,32 @@ Release Notes
 =============
 
 
+Release 1.1.0
+=============
+New functionality and features
+------------------------------
+* Merged TSPE "Total spiking probability edges" into elephant, a tool for classifying excitatory and inhibitory synaptic effects using connectivity estimation (https://github.com/NeuralEnsemble/elephant/pull/560).
```
Comment (mdenker) [inline doc/release_notes.rst:10]: I think this should be written more from a user's perspective and less technical
```suggestion
* New method "Total spiking probability edges" (TPSE) for inferring functional connectivity (https://github.com/NeuralEnsemble/elephant/pull/560).
```

**Review by mdenker — COMMENTED** [review]
I am not quite sure why all the Jupyter notebooks are in this PR. Did something change?

File: doc/release_notes.rst (hunk 1 above)
Comment (Moritz-Alexander-Kern) [reply doc/release_notes.rst:10]: I agree.

**Comment by Moritz-Alexander-Kern** [comment]
> I am not quite sure why all the Jupyter notebooks are in this PR. Did something change?

Yes, the "new" Neo documentation changed the links to the api documentation, see for e.g.:

Old:  <https://neo.readthedocs.io/en/stable/core.html>
New: <https://neo.readthedocs.io/en/stable/api_reference.html>

However that is just a detail, the notebooks themselves are unchanged.

**Review by mdenker — APPROVED** [review]

**Pull request merged**